frontend/src/components/Experiment/Experiment.test.js

```diff
@@ -10,45 +10,35 @@ jest.mock("../../util/stores");
 // creates a different object every time, causing useEffect to trigger unnecessarily
 const experimentObj = {
     id: 24, slug: 'test', name: 'Test', playlists: [{id: 42, name: 'TestPlaylist'}],
-    next_round: [{view: 'PLAYLIST'}]
+    next_round: [{view: 'INFO', button_label: 'Continue'}]
 };
 const sessionObj = {data: {session: {id: 1}}};
 const nextRoundObj = {next_round: [{view: 'EXPLAINER'}]};
 
 jest.mock("../../API", () => ({
-    useExperiment: () => [experimentObj, false],
+    useExperiment: () => {
+
+        return [experimentObj, false]
+    },
     createSession: () => Promise.resolve(sessionObj),
     getNextRound: () => Promise.resolve(nextRoundObj)
 }));
```

Comment:
As mentioned in my comment, perhaps it would be better and easier to mock Axios requests and return the data that would normally initialise the experiment in the correct / desired way.

Something like this:

```js
import axios from 'axios';

jest.mock('axios');

axios.get.mockResolvedValue({ data: {} }); // Adjust as needed
```

Reply:
I had been thinking along the same lines in the meantime. :-) Will try that.